controller.go

```diff
@@ -168,25 +186,28 @@ func (c *controller) SyncInformerCache() {
 // start the controller
 func (c *controller) run(stopch <-chan struct{}) {
 	Logger.Info("Started notification controller")
-	defer c.queue.ShutDown()
+	defer c.nsqueue.ShutDown()
+	defer c.backupqueue.ShutDown()
 
 	c.dynInformer.Start(stopch)
 	go c.nsinformer.Run(stopch)
 
-	go wait.Until(c.worker, 1*time.Second, stopch)
+	go wait.Until(c.worker1, 1*time.Second, stopch)
+
+	go wait.Until(c.worker2, 1*time.Second, stopch)
```

Comment:
let's name the worker something meaningful, like it is for queues. (crworker, nsworker may be)